source/math/FloatRandomGenerator.ooc

```diff
@@ -50,14 +58,16 @@ FloatGaussianRandomGenerator: class extends FloatRandomGenerator {
 	_backend: FloatUniformRandomGenerator
 	_mu := 0.0f
 	_sigma := 1.0f
+	_secondValue : Float
+	_hasSecond := false
```

Comment:
Actually, what do you even mean by "second value"? With a single value, you will generate numbers "around" that value, and with a second value, you will generate numbers in the range between them, is that it?